Add Safe and Multicall tools (#1537)
iotaledger/iota-wiki: docs/build/getting-started/safe.md

---
description: 'Multisig wallet refers to having multiple private keys for a single web3 wallet.'
image: /img/logo/WASP_logo_dark.png
---

# Safe Wallet

> ### Safe wallet is IOTA's exclusive [MultiSig](https://safe.iotaledger.net/welcome) wallet. The term “multisig” is short for "multisignature".

Reviewer: ```suggestion
:::note TL;DR;

Safe Wallet is IOTA's exclusive [MultiSig](https://safe.iotaledger.net/welcome) wallet. The term “
  multisig” is short for "multisignature".
  
:::
```

> The security of cryptocurrency demands greater attention as digital assets gain traction.  Individuals and organizations need robust strategies to safeguard their funds. Multisignature (multisig) wallets are emerging as a powerful cybersecurity tool in the crypto space.

Reviewer: ```suggestion
The security of cryptocurrency demands greater attention as digital assets gain traction. Individuals and organizations need robust strategies to safeguard their funds. Multisignature (multisig) wallets are emerging as a powerful cybersecurity tool in crypto.
```

> Unlike traditional finance, decentralized digital assets require a different approach to security. Thankfully, innovative technologies like multisig wallets provide an extra layer of protection, allowing you to trade cryptocurrencies on chains like [Shimmer EVM](https://explorer.evm.shimmer.network/) or [IOTA](https://explorer.evm.iota.org/).

Reviewer: ```suggestion
Decentralized digital assets require a different approach to security than traditional finance. Thankfully, innovative technologies like multisig wallets provide an extra layer of protection, allowing you to trade cryptocurrencies on chains like [Shimmer EVM](https://explorer.evm.shimmer.network/) or [IOTA](https://explorer.evm.iota.org/).
```

![IOTA MultiSig](/icons/iota/safe.png)

> ## Why are multisig wallets more secure for crypto?

Reviewer: ```suggestion
## Why Are Multisig Wallets More Secure?

```

Traditional crypto wallets rely on a single signature for transactions, making them vulnerable if that key is compromised. Multisig wallets offer superior security by requiring multiple approvals before funds can be moved. This added layer of verification significantly reduces the risk of unauthorized transactions, even if a hacker gains access to one of your keys.

> 

Reviewer: ```suggestion
```

> ## Use Cases for multisig wallet

Reviewer: ```suggestion
## Use Cases for Multisig Wallet
```

> ### Shared Asset Security:

Reviewer: ```suggestion
### Shared Asset Security

```

> - Joint Ownership: Securely manage crypto held jointly by couples, business partners, or organizations. Requires approval from all parties for transactions, preventing unauthorized spending.

Reviewer: ```suggestion
#### Joint Ownership

Securely manage crypto held jointly by couples, business partners, or organizations. Requires approval from all parties for transactions, preventing unauthorized spending.
```

> - Company Funds: For businesses with substantial crypto holdings, enforce multi-approval workflows for withdrawals, minimizing internal theft risk.

Reviewer: ```suggestion
#### Company Funds

For businesses with substantial crypto holdings, enforce multi-approval workflows for withdrawals, minimizing internal theft risk.
```

> ### Enhanced Threat Protection:

Reviewer: ```suggestion
### Enhanced Threat Protection

```

> - Lost/Compromised Keys: Mitigate the impact of lost or compromised keys. Remaining keys can still safeguard funds, unlike single-signature wallets.

Reviewer: ```suggestion
#### Lost/Compromised Keys

Mitigate the impact of lost or compromised keys. Remaining keys can still safeguard funds, unlike single-signature wallets.
```

> - Hacking Defense: Add a significant hurdle for attackers. Even with access to one key, they cannot steal funds without additional signatures.

Reviewer: ```suggestion
#### Hacking Defense

Add a significant hurdle for attackers. They cannot steal funds without additional signatures even with access to one key.
```

> - Custodial Account Control: Maintain control over crypto held in exchanges or IRA providers. Multisig wallets ensure withdrawals require approval from both the - user and the custodian, preventing unauthorized access.

Reviewer: ```suggestion
#### Custodial Account Control

Maintain control over crypto held in exchanges or IRA providers. Multisig wallets ensure withdrawals require approval from both the user and the custodian, preventing unauthorized access.
```

> ### Important Considerations:

Reviewer: ```suggestion
### Important Considerations

```

> - Technical Complexity: Setting up and using multisig wallets requires more technical expertise compared to traditional wallets.

Reviewer: ```suggestion
#### Technical Complexity

Setting up and using multisig wallets requires more technical expertise than traditional wallets.
```

> - Signatory Balance: Finding the optimal balance between security (more signatories) and practicality (fewer signatories) is crucial.

Reviewer: ```suggestion
#### Signatory Balance

Finding the optimal balance between security (more signatories) and practicality (fewer signatories) is crucial.
```

> - Individual Suitability: For most individual users with smaller crypto holdings, a standard single-signature wallet might be sufficient.

Reviewer: ```suggestion
#### Individual Suitability

A standard single-signature wallet might be sufficient for most individual users with smaller crypto holdings.
```

:::tip

> Learn more about safe account [here](https://help.safe.global/en/articles/40835-what-safe-setup-should-i-use).

Reviewer: ```suggestion
Learn more about safe accounts in the [Safe documentation](https://help.safe.global/en/articles/40835-what-safe-setup-should-i-use).
```

:::

> :::info

Reviewer: ```suggestion
:::info Set Up a Safe Account
```

> Setting up safe account: [here](https://help.safe.global/en/articles/40868-creating-a-safe-on-a-web-browser).

Reviewer: ```suggestion
You can find instructions on setting up a safe account in the [Safe documentation](https://help.safe.global/en/articles/40868-creating-a-safe-on-a-web-browser).
```

---
description: 'Multisig wallet refers to having multiple private keys for a single web3 wallet.'
image: /img/logo/WASP_logo_dark.png
---

# MultiSig Wallet

> :::note TL;DR;

Author: ```suggestion
:::note TL;DR;

```

> Safe Wallet is IOTA's exclusive [MultiSig](https://safe.iotaledger.net/welcome) wallet. The term “

Author: Why exclusive btw? Doesn't that mean it only is available on IOTA? 😅

Reviewer 2: Exclusivity in regards to "safe" on IOTA.
Yes, this safe is deployed on other chains also.

Author: Got it

  multisig” is short for "multisignature".

:::

The security of cryptocurrency demands greater attention as digital assets gain traction. Individuals and organizations need robust strategies to safeguard their funds. Multisignature (multisig) wallets are emerging as a powerful cybersecurity tool in crypto.

> Decentralized digital assets require a different approach to security than traditional finance. Thankfully, innovative technologies like multisig wallets provide an extra layer of protection, allowing you to trade cryptocurrencies on chains like [Shimmer EVM](https://explorer.evm.shimmer.network/) or [IOTA](https://explorer.evm.iota.org/).

Author: ```suggestion
Decentralized digital assets require a different approach to security than traditional finance. Thankfully, innovative technologies like multisig wallets provide an extra layer of protection, allowing you to trade cryptocurrencies on chains like [Shimmer EVM](https://explorer.evm.shimmer.network/) or [IOTA EVM](https://explorer.evm.iota.org/).
```

![IOTA Sandbox](/img/get-started/safe/safe.png)

> ## Why Are Multisig Wallets More Secure?

Author: ```suggestion
## Why Are Multisig Wallets More Secure?

```

### Enhanced Threat Protection

#### Lost/Compromised Keys

 Mitigate the impact of lost or compromised keys. Remaining keys can still safeguard funds, unlike single-signature wallets.

#### Hacking Defense

 Add a significant hurdle for attackers. Even with access to one key, they cannot steal funds without additional signatures.

#### Custodial Account Control

 Maintain control over crypto held in exchanges or IRA providers. Multisig wallets ensure withdrawals require approval from both the - user and the custodian, preventing unauthorized access.

### Important Considerations

#### Technical Complexity

 Setting up and using multisig wallets requires more technical expertise compared to traditional wallets.

#### Signatory Balance

Finding the optimal balance between security (more signatories) and practicality (fewer signatories) is crucial.

#### Individual Suitability

 For most individual users with smaller crypto holdings, a standard single-signature wallet might be sufficient.

:::tip

Learn more about safe accounts in the [Safe documentation](https://help.safe.global/en/articles/40835-what-safe-setup-should-i-use).

:::

> ## IOTA has its own dedicated MultiSig wallet - [safe(Wallet)](https://safe.iotaledger.net/welcome)
> :::info Set Up a Safe Account
> You can find instructions on setting up a safe account in the [Safe documentation](https://help.safe.global/en/articles/40868-creating-a-safe-on-a-web-browser).
> :::

Author: ```suggestion
## IOTA has its own dedicated MultiSig wallet - [safe(Wallet)](https://safe.iotaledger.net/welcome)

:::info Set Up a Safe Account

You can find instructions on setting up a safe account in the [Safe documentation](https://help.safe.global/en/articles/40868-creating-a-safe-on-a-web-browser).

:::
```

---
description: 'Multisig wallet refers to having multiple private keys for a single web3 wallet.'
image: /img/logo/WASP_logo_dark.png
---

> # MultiSig Wallet

Reviewer: ```suggestion
# MultiSig Wallet

```

:::note TL;DR;

> Safe Wallet is a [MultiSig](https://safe.iotaledger.net/welcome) wallet. The term “

Reviewer: ```suggestion
[Safe Wallet](https://safe.iotaledger.net/welcome) is a multiSig wallet. The term “
```

  multisig” is short for "multisignature".

:::

The security of cryptocurrency demands greater attention as digital assets gain traction. Individuals and organizations need robust strategies to safeguard their funds. Multisignature (multisig) wallets are emerging as a powerful cybersecurity tool in crypto.

Decentralized digital assets require a different approach to security than traditional finance. Thankfully, innovative technologies like multisig wallets provide an extra layer of protection, allowing you to trade cryptocurrencies on chains like [Shimmer EVM](https://explorer.evm.shimmer.network/) or [IOTA EVM](https://explorer.evm.iota.org/).

![IOTA Sandbox](/img/get-started/safe/safe.png)

## Why Are Multisig Wallets More Secure?

Traditional crypto wallets rely on a single signature for transactions, making them vulnerable if that key is compromised. Multisig wallets offer superior security by requiring multiple approvals before funds can be moved. This added layer of verification significantly reduces the risk of unauthorized transactions, even if a hacker gains access to one of your keys.

> ## Use Cases for multisig Wallet

Reviewer: ```suggestion
## Use Cases for Multisig Wallet
```
